start_pgbouncer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+PGB_DIR=/etc/pgbouncer
+INI=$PGB_DIR/pgbouncer.ini
+USERLIST=$PGB_DIR/userlist.txt
+
+rm $INI
+rm $USERLIST
+
+cat <<- END > $INI
+    [databases]
+    $PGB_DATABASES
+
+    [pgbouncer]
+    listen_port = $PGB_LISTEN_PORT
+    listen_addr = $PGB_LISTEN_ADDR
+    auth_type = md5
+    auth_file = $USERLIST
+    logfile = $PGB_DIR/pgbouncer.log
+    pidfile = $PGB_DIR/pgbouncer.pid
+    admin_users = $PGB_ADMIN_USERS
```

Comment:
```suggestion
    [databases]
    $PGB_DATABASES

    [pgbouncer]
    listen_port = ${PGB_LISTEN_PORT:-6432}
    listen_addr = ${PGB_LISTEN_ADDR:-0.0.0.0}
    auth_type = md5
    auth_file = $USERLIST
    logfile = $PGB_DIR/pgbouncer.log
    pidfile = $PGB_DIR/pgbouncer.pid
    admin_users = ${PGB_ADMIN_USERS:-admin}
```

Reply:
Before this, you probably want to check that `$PBG_DATABASES` is not empty, and if it is, fail:

```bash
if [[ -z "${PGB_DATABASES:-}" ]]; then
  echo "Error: no databases specified in $PBG_DATABASES" >2
  exit 1
fi